examples/model_compress/multi_gpu.py

```diff
@@ -0,0 +1,100 @@
+import torch
```

Comment:
suggest to leverage existing example for DataParallel, no need to add a new example.

Reply:
Sure. This example comes from origin issue.